Tests/SwiftStdlibTests/BinaryIntegerExtensionsTests.swift

```diff
@@ -0,0 +1,34 @@
+// BinaryIntegerExtensionsTests.swift - Copyright 2021 SwifterSwift
+
+@testable import SwifterSwift
+import XCTest
+
+final class BinaryIntegerExtensionsTests: XCTestCase {
+    func testBytes() {
+        let zero = Int32.zero.bytes
+        XCTAssertEqual(zero, Array(repeating: 0, count: 4))
+
+        let negativeOne = Int8(-1).bytes
+        assert(negativeOne == [0xFF])
```

Comment:
I think @guykogus comments about `XCTAssertEqual` are applicable to all `assert`s